src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure {
+    private int base;
+    private int height;
+
+    public RightTriangle() {
+        this.base = new Random().nextInt(15);
+        this.height = new Random().nextInt(15);
```

Comment:
lets init values outside
exact FigureSupplier should generate figures with random properties:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/ce46f7a7-b894-46cb-8d3f-d171679c653c)
```suggestion
```